docs/en/solutions/Cloud native technology copy.md

```
@@ -0,0 +1,45 @@
+   - Solution
+   - Test
+---
+
+# Cloud native technology: the core architecture for building modern applications
+
+![Schematic diagram of cloud native technology stack](../assets/ymyzd.png)  
+
+## 📌 What is cloud native?
+Cloud Native is a methodology for building and running applications that fully utilizes the advantages of the cloud computing delivery model to achieve elastic scaling, high availability, and fast iteration. It includes four pillars: containerization, microservices, DevOps, and continuous delivery.
+
+###Key features
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading

(MD018, no-missing-space-atx)